components/hooks/feeds/useCommunityFeed.ts

```diff
@@ -46,20 +45,23 @@ const useCommunityFeed = (communityFullName: string): UseCommunityFeed => {
   const onSubscribePress = () => {
     trigger("impactMedium");
 
-    toast.show({
-      title: `${!feed.subscribed ? "Subscribed to" : "Unsubscribed from"} ${
-        feed.community.community.name
-      }`,
-      duration: 3000,
-    });
-
     dispatch(
       subscribeToCommunity({
         communityId: feed.community.community.id,
         subscribe: !feed.subscribed,
       })
     );
 
+    dispatch(
```

Comment:
Probably better if we have the toast show up after successfully subscribing. We can do this in https://github.com/gkasdorf/memmy/blob/main/slices/communities/communitiesActions.ts. That way if it fails the user will know.